pkg/search/postgres/joins_test.go

```diff
@@ -0,0 +1,613 @@
+//go:build sql_integration
+// +build sql_integration
```

Comment:
Why do we need this?

Reply:
Tbh. I'm not sure. I have seen that tag used in other tests in this package. I assume it's related to group `go-postgres-unit-tests`.

Reply:
But you are not calling external database. I think we can remove it.